org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLWorkspaceService.java

```diff
@@ -44,9 +45,15 @@ public void didChangeConfiguration(DidChangeConfigurationParams params) {
 
 	@Override
 	public void didChangeWatchedFiles(DidChangeWatchedFilesParams params) {
-
+		saveChangeWatchedFiles(params);
 	}
 
-	
+	private void saveChangeWatchedFiles(DidChangeWatchedFilesParams params) {

```

Comment:
there's no added value in providing this private method. Might as well move the code directly under didChangeWatchedFiles